en/config_fw/position_tuning_guide_fixedwing.md

```diff
@@ -1,13 +1,8 @@
-# Advanced Fixed-wing Position Tuning
+# Fixed-wing Position Tuning
 
 This guide offers some help in tuning the high-level fixed-wing controllers needed for flying missions and in altitude/position control mode.
```

Comment:
For all docs we need to think who is our audience: how did they get here, how do they know it is relevant, and where are they going. So when I see this statement I think:

- When do I do this - always after PID tuning? Sometimes?
- What vehicle does it apply to? Are there particular frames it is less useful or necessary for?
- Are there particular use cases that really need this done?
- How do I decide I need to do this - symptoms of bad tuning, borderline tuning, OK tuning
- After doing this, how can I verify it worked? How do I know it is good enough to stop.

I know this is out of scope for your changes, but it's a big flaw in our current docs.

Reply:
Valid input. I think we should recommend to do this tuning for every newly brought up vehicle. After the rate/attitude PID tuning. 
I would also here propose to address this once we settled on the docs structure. The whole FW tuning guide can then be streamlined.